src/routers/project.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+const projectController = require("../controllers/project");
```

Comment:
here in the other part of the file we used project instead of projectController so here also we should go with the same naming